probe-rs/targets/SAMD21.yaml

```diff
@@ -892,7 +892,7 @@ flash_algorithms:
       program_page_timeout: 0x64
       erase_sector_timeout: 0x3e8
       sectors:
-        - size: 0x800
+        - size: 0x100
```

Comment:
What's the reason for this?

Reply:
The reason for this is that I stupidly added it with `-a`. I made these changes to the logs while debugging the ATSAMD21 :) I will remove those changes :)